py/picca/data.py

```diff
@@ -221,6 +221,11 @@ def __add__(self,d):
             ccnew = sp.bincount(bins, weights=iv * v)
             cnew[:len(ccnew)] += ccnew
             setattr(self, k, cnew[w] / civ[w])
+        err = 1./sp.sqrt(self.iv)
+        SNR = self.fl/err
+        self.mean_SNR = sp.mean(SNR)
+        lam_lya = constants.absorber_IGM["LYA"]
+        self.mean_z = (sp.power(10.,ll[len(ll)-1])+sp.power(10.,ll[0]))/2./lam_lya -1.0
 
```

Comment:
in between you added and removed the mean_reso, was there a reason for the removal?

Reply:
I was wrong to remove the mean_reso update. It's fixed now, thanks.